### PR TITLE
Check data visualization in ggplot2 module

### PR DESCRIPTION
**Pull request by rosemm**
Bringing ggplot2 module up to code with new module template and new lesson prep section without pangeobinder

### CODE REVIEW

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md [hunk 1]
```diff
@@ -18,7 +22,58 @@ After completion of this module, learners will be able to:
...
+		$this.toggleClass('play');
+
+	});
+
+})(jQuery);
+</script>
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+script: https://kit.fontawesome.com/83b2343bd4.js
```
Comment (leemc-data-ed) [inline data_visualization_in_ggplot2/data_visualization_ggplot2.md:76]: Also need to add - 
script:  https://code.jquery.com/jquery-3.6.0.slim.min.js

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md [hunk 2]
```diff
@@ -42,31 +97,84 @@ This module also assumes some basic familiarity with R, including
...
+* Select Version Control, then Git
+* Drop this link into the "Repository URL": https://github.com/arcus/education_r_environment
+* Change the "Project directory name" and "Create project as a subdirectory of" boxes to suit your needs (where will this code be stored on your computer?).
+* Click to select the "Open in new session" checkbox
+* Click "Create Project"
+* In the file area to the lower right, you'll see, among multiple choices, the folder called "@r_code".  That's the code for this module!
+
+**Want to watch this process?  Click on the image below to play an animated gif.  It will continue to loop and you can re-start it by clicking again.**
+
+<div style="display:none">@gifPreload</div>
```
Comment (leemc-data-ed) [inline data_visualization_in_ggplot2/data_visualization_ggplot2.md:151]: This GIF isn't rendering, but I think the extra line in the YAML should make it work.

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md [hunk 3]
```diff
@@ -92,13 +200,13 @@ At first, this seems like more work than just using a single command in another
 To learn more about the theory behind ggplot2, read [Hadley Wickham's article, "A Layered Grammar of Graphics"](http://vita.had.co.nz/papers/layered-grammar.pdf)
 </div>
 
-## Working through interactive coding examples
+## Working through interactive coding examples in binder
 
-Hopefully your [binder instance](#lesson-preparation) is done loading now! If not, be patient --- it can take as long as 20 or 30 minutes some times if the files haven't been used recently.
+If you chose [Option 1 in Lesson Preparation](#lesson-preparation) hopefully your binder instance is done loading now! If not, be patient --- it can take as long as 20 or 30 minutes some times if the files haven't been used recently.
```
Comment (leemc-data-ed) [inline data_visualization_in_ggplot2/data_visualization_ggplot2.md:205]: This internal link doesn't work.

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md [hunk 4]
```diff
@@ -42,31 +97,84 @@ This module also assumes some basic familiarity with R, including
...
-You don't have to do anything except come back here after opening the link opens in a new tab or window.
 
 ## Making plots in ggplot2
 
 This module is a practical, hands-on guide to making data visualizations in R's ggplot2. Snippets of code are included throughout the text here, but you are strongly encouraged to try running the code yourself instead of just reading it. Better yet, try to modify the code for each of the example plots to use with your own data!
 
-<div class = "warning">
-If you are using the [pangeo binder instance we prepared](#lesson-preparation), then all of the R packages you need will already be installed and you're all set.
+<div class = "important">
+If you are using the [binder instance we prepared](#lesson-preparation), then all of the R packages you need will already be installed and you're all set.
```
Comment (leemc-data-ed) [inline data_visualization_in_ggplot2/data_visualization_ggplot2.md:177]: This internal link also doesn't work.

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md [hunk 5]
```diff
@@ -134,7 +242,7 @@ breast_cancer_data <- read_csv("https://archive.ics.uci.edu/ml/machine-learning-
 <div class="learnmore">
 Run the above code yourself in binder (see [lesson preparation](#lesson-preparation) for links to start the binder instance) or on your own computer.
```
Comment (leemc-data-ed) [inline data_visualization_in_ggplot2/data_visualization_ggplot2.md:243]: Same problem with internal link to Lesson Preparation

**Review by leemc-data-ed — CHANGES_REQUESTED** [review]
Just a couple of little things! Just some broken internal links and the GIF not rendering. Otherwise lgtm!

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md (hunk 2 above)
Comment (leemc-data-ed) [reply data_visualization_in_ggplot2/data_visualization_ggplot2.md:151]: Or possibly it's because the url is pointing to main?

Comment (rosemm) [reply data_visualization_in_ggplot2/data_visualization_ggplot2.md:151]: Yeah, I pointed it to what will eventually be the correct link, but it means it doesn't render properly now...

File: data_visualization_in_ggplot2/data_visualization_ggplot2.md (hunk 1 above)
Comment (rosemm) [reply data_visualization_in_ggplot2/data_visualization_ggplot2.md:76]: Good catch, thank you!

**Comment by rosemm** [comment]
Okie dokie, I think it's all fixed! @leemc-data-ed

**Pull request merged**